cpu/riscv_common/include/periph_cpu_common.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup         cpu_riscv_common
+ * @{
+ *
+ * @file
+ * @brief           CPU specific definitions for internal peripheral handling
+ *
+ * @author          Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef PERIPH_CPU_COMMON_H
+#define PERIPH_CPU_COMMON_H
+
+#include <inttypes.h>
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Power management configuration
+ * @{
+ */
+#define PROVIDES_PM_SET_LOWEST
+/** @} */
+
+/**
+ * @brief   Length of the CPU_ID in octets
+ */
+#define CPUID_LEN           (12U)
```

Comment:
Is this true for all RISC-V CPUs?
Or should that rather be moved to fe310

Reply:
Should probably be moved to the fe310, there is no common CPU_ID register on the RISC-V architecture. For the fe310, the vendor ID, the architecture ID and the implementation ID are concatenated. Not perfect and probably identical between different boards, but I haven't found a better ID in the mean time.

I'll move this to the fe310 header